packages/sources/css/src/components/button/src/index.css

```diff
@@ -164,19 +165,22 @@
 
   /* Sizes */
   .vtmn-btn_size--small {
-    font-size: 0.75rem;
-    padding: 0.5rem 1rem;
+    font-size: 0.875rem;
+    padding: 0.4375rem 1rem;
```

Comment:
Why this padding? 
I've followed https://www.decathlon.design/726f8c765/p/8008f8-button/b/50afe1

![image](https://user-images.githubusercontent.com/9600228/110679099-5e997880-81d7-11eb-90a9-d9869f02eacc.png)